user/plugins/random-shorturls/plugin.php

```diff
@@ -61,7 +61,13 @@ function ozh_random_shorturl_settings_page() {
     }
 
     $random_length = yourls_get_option('random_shorturls_length');
-    $nonce = yourls_create_nonce( 'random_shorturl_settings' );
+    $nonce         = yourls_create_nonce('random_shorturl_settings');
```

Comment:
Isn't this also auto-formatting?

Reply:
Thank you, keeps my honest. I did a few tests to make sure my ide will not auto-format anymore.